packages/aggregator/src/connections/DataSourceEntityProvider.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+
+export class DataSourceEntityProvider {
```

Comment:
Wasn't entirely sure what to name this class, but wanted to give it a sort of 'function oriented' name. Rather that it saying 'I'm basically a connection to the entity-server'  I wanted it to say 'here's the data I can provide to you'

Reply:
I love a bit of code personification.